Assets/Creatures/api/PlayableCreatureBase.cs

```diff
@@ -0,0 +1,81 @@
+using System.Collections.Generic;
+using Creatures.Api;
+using Creatures.api.abilities;
+using Creatures.api.abilities.basic;
+using Creatures.api.abilities.character;
+using Creatures.api.abilities.charged;
+using Creatures.api.abilities.infusion;
+using Creatures.api.abilities.mobility;
+using Creatures.api.abilities.states;
+
+namespace Creatures
+{
+    public abstract class PlayableCreatureBase : Creature, IPlayable
+    {        
+        public ICoreInfusion CoreInfusion { get; set; }
+        private readonly CooldownMediator _cooldownMediator = new CooldownMediator();
+        private IEnumerable<AbilityEvent> abilityEvents = new List<AbilityEvent>
```

Comment:
Is there an immutable list?

Reply:
I'll create an issue to fix this, since my next PR is based on this one and you weren't blocking merge 🙇